packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -387,10 +387,11 @@ flowchart LR
 ### Entity codes to escape characters
 
 It is possible to escape characters using the syntax exemplified here.
+For quotation marks, you can escape them with a backslash.
 
 ```mermaid-example
     flowchart LR
-        A["A double quote:#quot;"] -->B["A dec char:#9829;"]
+        A["A double quote:\""] -->B["A dec char:#9829;"]
```

Comment:
More examples on why we cannot add `\` as escape character implicitly
If a guy has a node
```
graph
node["dir\"]
```

```mermaid
graph
node["dir\"]
```
This node will become invalid in our new approach

And if we make our users to escape `\` with `\` then
```
graph
node["dir\\"]
```

```mermaid
graph
node["dir\\"]
```
becomes invalid too, because in that case user would have to write
```
node["dir\\\\"]
```
to get the same result

Reply:
May be @sidharthv96 or @knsv can enlighten us on making breaking changes, so I'd invite them in this discussion. I think we are pretty conservative of such changes because of the size of community so we are obliged to support many undesirable features for the backward compatibility

Reply:
This is kind of still a breaking change, but what we might be able to do is use single quotes to act as literal strings where nothing will be escaped and everything shows up as is. So, if users wanted slashes or to write `\n` then they would use single quotes instead of double quotes

Reply:
Well, in this branch a string like `node["dir\"]` would break because `\"` would be read as an escaped double quote, but I don't think something like `node["http:\\ link"]` would be a problem. Can't we just tell users to use the HTML code `#bsol;` if they need a backslash at the end of the string? Because creating a non-breaking solution will probably lead to other undesirable effects no?

Reply:
Really, I can remove the escaped quotes feature, because that wasn't there before. That was added based on @sidharthv96 's [table above](https://github.com/mermaid-js/mermaid/pull/4657#issuecomment-1646660045)

Reply:
We can extend existing behavior, but cannot change previous one, unless it is a bug. `node["dir\"]` is expected to produce node with `dir\` label by our users, so we cannot change that. It have to keep working. There was only code-like quotes allowed inside a string, so we can ask to continue using it, in case one need a quote

Reply:
Sadly, breaking changes are only possible for our code, not syntax. It's frustrating, but we cannot have old diagrams breaking in sites after a mermaid upgrade. 

I didn't consider the end quote problem when adding the table, nice catch there @nirname. I guess we'll have to stick with `#quot;` for now, maybe until we come up with a better grammar overall, with fallback support to old grammars.